cmd/fluxctl/install/install.go

```diff
@@ -0,0 +1,54 @@
+package install
```

Comment:
This should probably go under the top directory, if you want to use it from elsewhere.

Reply:
makes sense